frontend/components/dashboard/table/DashboardTableSummary.vue

```diff
@@ -328,6 +328,7 @@ const searchPlaceholder = computed(() =>
             </template>
             <template #empty>
               <DashboardTableAddValidator v-if="!hasValidators" />
+              <DashboardTableEmpty v-else />
```

Comment:
Please discuss with Product Owner if it would make sense to add this empty message to all tables as default empty message. 

If not in general then at least for all Dashboard (Validator and Notfications) page tables

Reply:
For now we want to display the message for any empty table.